common/scripts/generate-doc.js

```diff
@@ -200,7 +200,8 @@ for (const eachService of serviceFolders) {
                   if (checks.typedocPresent) {
                     const typedocResult = childProcess.spawnSync(
                       "typedoc",
-                      [docOutputFolder],
+                      [docOutputFolder,
+                      "--ignoreCompilerErrors"],
```

Comment:
@HarshaNalluru @jeremymeng Do we know why this got added? This is not related directly to storage

Reply:
It is fine. It is added to make typedoc work for storage after I added typedoc.json

Reply:
I added typedoc.json config files for storage. This was needed to fix the failure of generating the docs.

Reply:
Our other libraries don't use typedoc.json?

Reply:
Logged #5596